ext/bcmath/libbcmath/src/num2str.c

```diff
@@ -73,6 +73,6 @@ zend_string *bc_num2str_ex(bc_num num, size_t scale)
 
 	/* Terminate the string and return it! */
 	*sptr = '\0';
-	ZSTR_LEN(str) = sptr - (char *) ZSTR_VAL(str);
-	return str;
+	ZSTR_LEN(str) = sptr - ZSTR_VAL(str);
+	return (zend_string*)str;
```

Comment:
Because my IDE shows the warning: `Incompatible pointer types returning 'char *' from a function with result type 'int *'`

However I can't fix the Travis CI. Do you have a hint how to reproduce this on my local env to fix it there?